Tests/Form/Type/BooleanTypeTest.php

```diff
@@ -85,11 +85,11 @@ public function testOptions()
         ), $resolvedOptions);
     }
 
-    public function testDeprecatedCatalogueOption()
+    public function testLegacyCatalogueOption()
```

Comment:
Why use `configureOptions` and rename this method to legacy?

IMO, both tests should be here.